stable/jenkins/master-image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jenkins:2.32.3
-RUN /usr/local/bin/install-plugins.sh kubernetes:0.11 workflow-aggregator:2.5 credentials-binding:1.10 git:3.1.0 \
+FROM jenkinsci/jenkins:2.53
```

Comment:
Generally speaking no one runs the latest weekly release in production and IMO this chart should sticky to the latest lts release which at the time of this comment is `2.46.1`. 

https://jenkins.io/changelog-stable/